src/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+  font-weight: 300px;
+}
+
+.search-bar-big {
+  box-sizing: border-box;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: 100%;
+  font-size: 16px;
+  padding: 25px 62px 23px;
+}
+
+.search-bar-small {
+  box-sizing: border-box;
+  display: block;
+
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+  padding: 12px 33px 10px;
+}
+
```

Comment:
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 8th item, you should style `::placeholder` for inputs and `:focus` state